framework/src/modules/interoperability/schema.ts

```diff
@@ -429,6 +429,39 @@ export const crossChainUpdateTransactionParams = {
 	},
 };
 
+export const messageRecoveryParams = {
+	$id: '/modules/interoperability/mainchain/messageRecovery',
+	type: 'object',
+	required: ['chainID', 'crossChainMessages', 'idxs', 'siblingHashes'],
+	properties: {
+		chainID: {
+			dataType: 'uint32',
+			fieldNumber: 1,
+		},
+		crossChainMessages: {
```

Comment:
Probably the `minItems` field is missing as per this [comment](https://github.com/LiskHQ/lisk-sdk/pull/7142#discussion_r869395118)